src/MapboxStyleParser.ts

```diff
@@ -434,27 +475,28 @@ export class MapboxStyleParser implements StyleParser<Omit<MbStyle, 'sources'>>
    * @param layout A Mapbox layout
    * @return A GeoStylerStyle-FillSymbolizer
    */
-  getFillSymbolizerFromMapboxFillLayer(paint: FillPaint, layout: FillLayout): FillSymbolizer {
+  async getFillSymbolizerFromMapboxFillLayer(paint: FillPaint, layout: FillLayout): Promise<FillSymbolizer> {
+    const graphicFill = await this.getPatternOrGradientFromMapboxLayer(paint?.['fill-pattern']);
```

Comment:
See comment above

Reply:
I did not really change the code here, so this is not part of this PR. But we might want to add some more complex testdata to see where we have some gaps left.